app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -175,6 +176,8 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
      */
     private var playerLayoutOrientation = Int.MIN_VALUE
 
+    private var pipActivity: Activity? = null
```

Comment:
Please add a short comment explaining what this is used for for future reference. (e.g. `Activity that's active during PiP, can be used for controlling its lifecycle`)